Automattic/jetpack
# docker: Add script to create "fake" translations for testing i18n

#### Changes proposed in this Pull Request:

This adds a script, intended to be run inside the Jetpack Docker
environment, to create plugin translations for two fake languages.

- en_piglatin: All messages are translated into Pig Latin.
- en_rtl: All messages will display backwards, and the interface will be
  right-to-left.

Once you have run the script, in WordPress Admin > Settings > General
you should see the above two languages as options for "Site Language".

Note WordPress itself is not translated as described, the translations
are only created for messages in the specified plugin.

#### Jetpack product discussion


p1634159370450900-slack-CBG1CP4EN

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Make sure you have the latest docker image, last I checked the one from Docker was outdated. You might need to `jetpack docker build-image`.
* Run `jetpack docker exec /var/scripts/fakepot.sh /var/www/html/wp-content/plugins/jetpack`
* Open your site to `/wp-admin/options-general.php`. You should see the two new languages in the Site Languages dropdown.
* If you select "en_piglatin" and go to the Jetpack dashboard, you should see something like this: 
![en_piglatin screenshot](https://user-images.githubusercontent.com/1030580/137212575-c67b66d6-88ab-4dd9-957c-3394d6f482a2.png)
* If you select "en_rtl" and go to the Jetpack dashboard, you should see something like this: 
![en_rtl screenshot](https://user-images.githubusercontent.com/1030580/137212627-4239d384-bcea-4eda-ad8f-5a66c90a8d53.png)
